app/_src/introduction/kuma-requirements.md

```diff
@@ -10,20 +10,37 @@ This page exposes the different requirements to run {{site.mesh_product_name}}.
 
 ## Kubernetes
 
-{{site.mesh_product_name}} is validated against Kubernetes `1.27.x`, `1.23.x`. For the specific Kubernetes version refer to the {{site.mesh_product_name}} [Makefile](https://github.com/kumahq/kuma/blob/master/mk/dev.mk#L24-L25).
+{{site.mesh_product_name}} is validated against Kubernetes versions:
```

Comment:
```suggestion
{{site.mesh_product_name}} is validated against two Kubernetes versions:
```

otherwise not 100% clear if it's some or all in between